.github/workflows/build-gtfs-rt-parser.yml

```diff
@@ -0,0 +1,27 @@
+name: Docker image
+
+on:
+  push:
+    branches:
+      - 'master'
```

Comment:
@atvaccaro Should this be `main`?

Reply:
I guess https://github.com/cal-itp/data-infra/issues/1304 refers to this.

Reply:
oops yes, thanks; resolving in PR